client/teleports.lua

```diff
@@ -37,7 +35,6 @@ CreateThread(function()
                                     SetEntityHeading(ped, Config.Teleports[loc][1].coords.w)
                                 end
                             end
-                            ResetTeleport()
```

Comment:
Why remove this?

Reply:
Because the function was setting a variable to true and that variable was never used by anything. So essentially a useless function and a useless variable got removed